### PR TITLE
To update the description for the Adaptive response notable events resource module

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

To update the description for the Adaptive response notable events resource module

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

splunk_adaptive_response_notable_events.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/fix_adaptive_response_ne_description.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
```
Comment (NilashishC) [inline changelogs/fragments/fix_adaptive_response_ne_description.yaml:2]: doc_changes:
```

File: plugins/modules/splunk_adaptive_response_notable_events.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 DOCUMENTATION = """
 ---
 module: splunk_adaptive_response_notable_events
-short_description: Manage Splunk Enterprise Security Notable Event Adaptive Responses
+short_description: Manage Adaptive Responses notable events resource module
```
Comment (NilashishC) [inline plugins/modules/splunk_adaptive_response_notable_events.py:14]: Could you also update the doc RST?

File: changelogs/fragments/fix_adaptive_response_ne_description.yaml (hunk 1 above)
Comment (justjais) [reply changelogs/fragments/fix_adaptive_response_ne_description.yaml:2]: updated

File: plugins/modules/splunk_adaptive_response_notable_events.py (hunk 2 above)
Comment (justjais) [reply plugins/modules/splunk_adaptive_response_notable_events.py:14]: will generate in release prep PR

File: changelogs/fragments/fix_adaptive_response_ne_description.yaml (hunk 1 above)
Comment (justjais) [reply changelogs/fragments/fix_adaptive_response_ne_description.yaml:2]: ERROR: Found 1 changelog issue(s) which need to be resolved:
--
1149 | 2022-09-06 14:52:05.437442 \| controller \| ERROR: changelogs/fragments/fix_adaptive_response_ne_description.yaml:0:0: invalid section: doc_changes

**Pull request merged**